CI unit tests fail to run typedoc-theme.js

Repro:
```sh
rvm use 2.5.2
bundle exec rake test
```

Failure: https://github.com/sass/sass-site/actions/runs/4919660406/jobs/8787552930

```
ln -sf ../.language/node_modules ../tool/node_modules
npx typedoc             --plugin ../tool/typedoc-theme.js --theme sass-site             --out ../source/documentation/js-api             --cleanOutputDir             js-api-doc/index.d.ts
        
Info: Loaded plugin /home/goodwine/repos/sass-site/tool/typedoc-theme.js
Error: spec/js-api/legacy/function.d.ts:1:32 - error TS2307: Cannot find module './plugin_this' or its corresponding type declarations.

1 import {LegacyPluginThis} from './plugin_this';
                                 ~~~~~~~~~~~~~~~

Error: spec/js-api/legacy/render.d.ts:2:29 - error TS2307: Cannot find module './options' or its corresponding type declarations.

2 import {LegacyOptions} from './options';
                              ~~~~~~~~~~~
```

I don't think either sass/sass-site nor sass/sass changed in a way that would trigger this error 🤔. But maybe sass/sass did change recently?

--- a/Rakefile
+++ b/Rakefile
@@ -146,7 +146,7 @@ namespace :sass do
         sh %{npm run typedoc -- \
             --plugin ../tool/typedoc-theme.js --theme sass-site \
             --out ../source/documentation/js-api \
-            --cleanOutputDir \
+            --cleanOutputDir
         }
       end
       sh %{rm -r source/documentation/js-api/assets}

--- a/Rakefile
+++ b/Rakefile
@@ -143,11 +143,10 @@ namespace :sass do
       Dir.chdir(".language") do
         sh %{npm install}
         sh %{ln -sf ../.language/node_modules ../tool/node_modules}
-        sh %{npx typedoc \
+        sh %{npm run typedoc -- \
             --plugin ../tool/typedoc-theme.js --theme sass-site \
             --out ../source/documentation/js-api \
             --cleanOutputDir \
-            js-api-doc/index.d.ts
         }
       end
       sh %{rm -r source/documentation/js-api/assets}

